src/query/union/buffered_union.rs

```diff
@@ -81,6 +83,7 @@ impl<TScorer: Scorer, TScoreCombiner: ScoreCombiner> BufferedUnionScorer<TScorer
     pub(crate) fn build(
         docsets: Vec<TScorer>,
         score_combiner_fn: impl FnOnce() -> TScoreCombiner,
+        num_docs: u32,
```

Comment:
Can you add a comment explaining what num_docs is about? I really did not understand what we are supposed to pass here.